README.md

```diff
@@ -0,0 +1,161 @@
+# Node Label Operator
+
+## Purpose
+
+This is a PoC for an operator with the main purpose of adding labels to
+Nodes based on their names, immediately when Nodes are created. This is done
+by using a mutating admission webhook.
+
+This functionality is needed in case you can't use kubelet to add the labels
+directly, and no other component should see the Node without those labels.
+
+For being able to maintain the labels after node creation as well, the operator
+will also add and optionally modify and delete labels on existing nodes in case
+the configuration changes.
+
```

Comment:
Might be worth explicitly mentioning that Nodes can match with multiple CRs to accumulate multiple sets of labels.

Reply:
Added it to the Labels CRD section :+1: